docs/csharp/programming-guide/strings/index.md

```diff
@@ -1,156 +1,182 @@
+
+In this example, a <xref:System.Text.StringBuilder> object is used to create a string from a set of numeric types:
+
+:::code language="csharp" source="./snippets/StringCharacters.cs" id="TestStringBuilder":::
+
+## Strings, extension methods and LINQ
+
+Because the <xref:System.String> type implements <xref:System.Collections.Generic.IEnumerable%601>, you can use the extension methods defined in the <xref:System.Linq.Enumerable> class on strings. To avoid visual clutter, these methods are excluded from IntelliSense for the <xref:System.String> type, but they're available nevertheless. You can also use LINQ query expressions on strings. For more information, see [LINQ and Strings](../concepts/linq/linq-and-strings.md).
+
+## Related articles
+
+- [How to modify string contents](../../how-to/modify-string-contents.md): Illustrates techniques to transform strings and modify the contents of strings.
+- [How to compare strings](../../how-to/compare-strings.md): Shows how to perform ordinal and culture specific comparisons of strings.
+- [How to concatenate multiple strings](../../how-to/concatenate-multiple-strings.md): Demonstrates various ways to join multiple strings into one.
+- [How to parse strings using String.Split](../../how-to/parse-strings-using-split.md): Contains code examples that illustrate how to use the <xref:System.String.Split%2A?displayProperty=nameWithType> method to parse strings.
+- [How to search strings](../../how-to/search-strings.md): Explains how to use search for specific text or patterns in strings.
+- [How to determine whether a string represents a numeric value](./how-to-determine-whether-a-string-represents-a-numeric-value.md): Shows how to safely parse a string to see whether it has a valid numeric value.
+- [String interpolation](../../language-reference/tokens/interpolated.md): Describes the string interpolation feature that provides a convenient syntax to format strings.
+- [Basic String Operations](../../../standard/base-types/basic-string-operations.md): Provides links to articles that use <xref:System.String?displayProperty=nameWithType> and <xref:System.Text.StringBuilder?displayProperty=nameWithType> methods to perform basic string operations.
+- [Parsing Strings](../../../standard/base-types/parsing-strings.md): Describes how to convert string representations of .NET base types to instances of the corresponding types.
+- [Parsing Date and Time Strings in .NET](../../../standard/base-types/parsing-datetime.md): Shows how to convert a string such as "01/24/2008" to a <xref:System.DateTime?displayProperty=nameWithType> object.
+- [Comparing Strings](../../../standard/base-types/comparing.md): Includes information about how to compare strings and provides examples in C# and Visual Basic.
+- [Using the StringBuilder Class](../../../standard/base-types/stringbuilder.md): Describes how to create and modify dynamic string objects by using the <xref:System.Text.StringBuilder> class.
```

Comment:
i skipped most of this file.

Reply:
Thanks. Only the one section has been substantially changed.